ADAL/src/ADOAuth2Constants.m

```diff
@@ -94,4 +94,6 @@
 
 //application constants
 NSString* const ADAL_BROKER_SCHEME = @"msauth";
+NSString* const ADAL_BROKER_APP_REDIRECT_URI = @"urn:ietf:wg:oauth:2.0:oob";
```

Comment:
RPangrle suggested that we appropriate this URI for the Authenticator's use. Android Authenticator is already using this URI to call through broker.